src/Model/PhasorDynamics/Branch/Branch.hpp

```diff
@@ -51,21 +45,21 @@ namespace GridKit
       using Component<ScalarT, IdxT>::yp_;
       using Component<ScalarT, IdxT>::tag_;
       using Component<ScalarT, IdxT>::f_;
-      using Component<ScalarT, IdxT>::g_;
-      using Component<ScalarT, IdxT>::yB_;
-      using Component<ScalarT, IdxT>::ypB_;
-      using Component<ScalarT, IdxT>::fB_;
-      using Component<ScalarT, IdxT>::gB_;
-      using Component<ScalarT, IdxT>::param_;
+      // using Component<ScalarT, IdxT>::g_;
+      // using Component<ScalarT, IdxT>::yB_;
+      // using Component<ScalarT, IdxT>::ypB_;
+      // using Component<ScalarT, IdxT>::fB_;
+      // using Component<ScalarT, IdxT>::gB_;
+      // using Component<ScalarT, IdxT>::param_;

```

Comment:
why are these just commented out, is this incomplete? if so, it'd be better to add a note as to why. if they're unnecessary, then they should be deleted.